web/templates/edit_invite_user_modal.hbs

```diff
@@ -0,0 +1,6 @@
+<form id="edit-invite-form">
+    {{> user_roles_dropdown invite_as_options=invite_as_options is_admin=is_admin is_owner=is_owner }}
+    {{#if can_subscribe_other_users}}
```

Comment:
This is not correct. This setting checks whether user can subscribe others to all the streams, but user can also have permission to subscribe others to just a single stream. We should show the UI always.